--- origsrc/goocanvas-1.2.6/lib/goocanvas.rb	2013-08-05 05:28:17.371722700 -0500
+++ src/goocanvas-1.2.6/lib/goocanvas.rb	2013-08-05 05:28:39.266975000 -0500
@@ -4,7 +4,6 @@ require 'cairo'
 base_dir = Pathname.new(__FILE__).dirname.dirname.expand_path
 vendor_dir = base_dir + "vendor" + "local"
 vendor_bin_dir = vendor_dir + "bin"
-GLib.prepend_environment_path(vendor_bin_dir)
 begin
   major, minor, micro, = RUBY_VERSION.split(/\./)
   require "#{major}.#{minor}/goocanvas.so"
